packages/integration-tests/lambda.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const { sfn } = require('@cumulus/common/aws');
+
+const lambdaScheduleEvents = [
+  'LambdaFunctionScheduled',
+  'LambdaFunctionScheduleFailed'
+];
+
+const lambdaStartedEvents = [
+  'LambdaFunctionStartFailed',
+  'LambdaFunctionStarted'
+];
+
+const lambdaCompletedEvents = [
+  'LambdaFunctionFailed',
+  'LambdaFunctionSucceeded',
+  'LambdaFunctionTimedOut'
+];
+
+/**
+ * Get the events for the lambda execution for the given workflow execution.
+ * This function currently assumes one execution of the given lambda per workflow.
+ *
+ * @param {string} workflowExecutionArn - Arn of the workflow execution
+ * @param {string} lambdaName - name of the lambda
+ * @returns {Object} an object containing a schedule event, start event, and complete
+ * event if exist, null if cannot find the lambda
+ */
+async function getLambdaExecution(workflowExecutionArn, lambdaName) {
+  const executionHistory = (
```

Comment:
Why the parens around this?

Reply:
https://github.com/airbnb/javascript#variables--linebreak

Reply:
Bonus points for throwing the docs back at me.